mne/inverse_sparse/mxne_inverse.py

```diff
@@ -630,7 +630,12 @@ def tf_mixed_norm(evoked, forward, noise_cov, alpha_space, alpha_time,
     M = np.dot(whitener, M)
 
     # Scaling to make setting of alpha easy
-    alpha_max = norm_l2inf(np.dot(gain.T, M), n_dip_per_pos, copy=False)
+    n_steps = int(np.ceil(M.shape[1] / float(tstep)))
+    n_freqs = wsize // 2 + 1
+    n_coefs = n_steps * n_freqs
+    phi = _Phi(wsize, tstep, n_coefs)
+    l1_ratio = alpha_time / (alpha_space + alpha_time)
+    alpha_max = norm_epsilon_inf(gain, M, phi, l1_ratio, n_dip_per_pos)
```

Comment:
this changes the parametrization can you check the impact on the examples? do you see the same sources? same waveforms?

Reply:
I've increased alpha_space from 30 to 35 to preserve the output of the TFMxNE example